CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ See [GitHub releases](https://github.com/jenkinsci/jira-plugin/releases)
 ### Unreleased
 Release date:  _March 2, 2020_
```

Comment:
? we are now using https://github.com/jenkinsci/jira-plugin/releases

Reply:
@olamy I added CHANGELOG.md before we've started using GH releases, I'll take a look if we could combine both approaches (and I still think it's a good idea to have CHANGELOG directly in git) - we can do both for now I guess.